snuba/query/processors/logical/quota_processor.py

```diff
@@ -7,11 +7,22 @@
 
 ENABLED_CONFIG = "resource_quota_processor_enabled"
 REFERRER_PROJECT_CONFIG = "referrer_project_thread_quota"
+REFERRER_CONFIG = "referrer_thread_quota"
 
 
 class ResourceQuotaProcessor(LogicalQueryProcessor):
     """
-    Applies a referrer/project thread quota to the query.
+    Applies a referrer/project thread quota to the query. Can throttle a referrer
+    or a (referrer, project) pair. The more specific restriction takes precedence
```

Comment:
I wonder if precedence should be the minimum number of max threads. If `REFERRER_{X}` is capped at 2 threads globally, it doesn't seem like `REFERRER_{X}_PROJECT_{Y}` should be allowed to have 10.

Though in practice I'm guessing people won't tend to mess this up